README.md

```diff
@@ -1,17 +1,109 @@
 # Mira S3 Webhosting Sample Application
 
-This application is a very basic application, that provides ready to use project setup for [Mira accelerator].
+This application is a very basic application, that provides a ready to use project setup for the [Mira accelerator].
 
-The successful deployment of this project will create an S3 bucket with web hosting enabled and a single index.html file in it.
+The successful deployment of this project will create an S3 bucket with web hosting enabled and a single index.html file in it, which will output a 'hello world' message.
+
+This sample app is recommended primarily for evaluation purposes, and project owners who want the benefits of Mira CI orchestration but wish to make a clean start with their project.
 
 ## Getting Started
 
-1. Install dependencies with `npm install`
-2. Run `npx mira docs` and navigate to [http://localhost:3000/#/quick-start/](http://localhost:3000/#/quick-start/) for further instructions
+1. Generate your own project repository from this repo [mira-sample-s3-webhosting](https://github.com/nearform/mira-sample-s3-webhosting/generate)
+
+    ⚠️ Remember to make your new repository __🕵🏻‍♀️ Private__ as you store potentially sensitive information in the Mira configuration files.
```

Comment:
Could add: "This includes your account number and potentially bucket information that may otherwise be public if someone had access to your CDK-generated CloudFormation template files."

Reply:
Let's leave it simple, no need for the detail